README.md

```diff
@@ -13,26 +20,69 @@ Installing latest requires creating a new catalog source.
 1. Navigate to _Operators => OperatorHub_.
 1. Search for _Tackle_.
 1. There should be two _Tackle_ available for installation now.
-1. Select the _Tackle_ without the _community_ tag.
+1. Select the _Tackle_ **without** the _community_ tag.
 1. Proceed to install latest.
 
-## Tackle CR Creation
+### Installing _latest_ on Kubernetes (or Minikube)
+
+See [k8s.md](./docs/k8s.md) for details.
+
+## Tackle CR Creation on OpenShift
 
 1. Visit OpenShift Web Console, navigate to _Operators => Installed Operators_.
 1. Select _Tackle_.
 1. Locate _Tackle_ on the top menu and click on it.
 1. Adjust settings if desired and click Create instance.
 
-### Installing _latest_ on Kubernetes (or Minikube)
+**Note:** Please review storage requirements **prior** creating the Tackle CR in case you need to adjust settings.
 
-See [k8s.md](./docs/k8s.md) for details.
+## Tackle CR Settings
 
-## Customize Settings
+If operator defaults need to be altered, the Tackle CR spec can be customized to meet desired needs, see the table below for some of the most significant settings:
+
+Name | Default | Description
+--- | --- | ---
+hub_database_volume_size | 5Gi | Size requested for Hub database volume
+hub_bucket_volume_size | 100gi | Size requested for Hub bucket volume
+keycloak_database_data_volume_size | 1Gi | Size requested for Keycloak DB volume
+pathfinder_database_data_volume_size | 1Gi | Size requested for Pathfinder DB volume
+windup_data_volume_size | 100Gi | Size requested for Windup addon volume
+tackle_rwx_storage_class | N/A | Storage class requested for Tackle RWX volumes
+tackle_rwo_storage_class | N/A | Storage class requested for Tackle RWO volumes
```

Comment:
Since all of these variables are `tackle` operator settings, I wonder if tackle_ is useful.

Reply:
My rationale was for the tackle_ naming was to hint that the variables affected *all* RWX or RWO volumes, to answer the question specifically no I don't think we will ever see non-tackle related variables.

Reply:
Don't mean to be pedantic but I think the tackle_ prefix is unnecessary and mildly suggests that the other variables may not be tackle related.

Reply:
Removed unnecessary tackle_ prefixing to storage class variables